From: =?utf-8?q?Ond=C5=99ej_Sur=C3=BD?= <ondrej@debian.org>
Date: Sun, 18 Sep 2016 11:26:45 +0200
Subject: Split example config files

---
 doc/bird.conf.example | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/bird.conf.example b/doc/bird.conf.example
index bbfe002..44dfc53 100644
--- a/doc/bird.conf.example
+++ b/doc/bird.conf.example
@@ -10,7 +10,7 @@
 #log "tmp" all;
 
 # Override router ID
-#router id 198.51.100.1;
+router id 198.51.100.1;
 
 # You can define your own symbols...
 #define xyzzy = (120+10);
